src/pages/Home.tsx

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query';
+import { Profile, profileService } from '../services/ProfileService';
+import LoadingScreen from '@/components/Loading';
+
+export default function Home() {
+  const {
+    data: profile,
+    isLoading,
+    isError,
+  } = useQuery<Profile | undefined>({
+    queryKey: ['profile', 1],
+    queryFn: async () => {
+      const response = await profileService.getProfile(1);
+      return response;
+    },
+  });
+
+  return (
+    <div>
+      {isLoading ? (
+        <LoadingScreen />
+      ) : isError ? (
+        <div>Erro ao carregar</div>
+      ) : (
+        <div>
+          <h1>{profile?.title}</h1>
+          <p>{profile?.completed ? 'Completo' : 'Incompleto'}</p>
+        </div>
+      )}
+    </div>
+  );
+}
```

Comment:
I prefer using early returns in this case. The presence of nested ternaries suggests that early returns would be a better approach.

(don't click "Apply Changes" on git, I'll probably break the formatting because I wrote it without linting)

```suggestion
  if (isLoading) {
    return <LoadingScreen />
  }
  
  if (isError) {
    return <div>Erro ao carregar</div>
  }
  
  return (
        <div>
          <h1>{profile?.title}</h1>
          <p>{profile?.completed ? 'Completo' : 'Incompleto'}</p>
        </div>
  );
}
```